pyaedt/modeler/circuits/object3dcircuit.py

```diff
@@ -681,12 +682,17 @@ def angle(self):
                 if "Angle=" in info:
                     self._angle = float(info[6:])
                     break
-        else:
+        elif settings.aedt_version > "2023.2":
             self._angle = float(
                 self._oeditor.GetPropertyValue("BaseElementTab", self.composed_name, "Component Angle").replace(
                     "deg", ""
                 )
             )
+        else:  # pragma: no cover
+            self._circuit_components._app.logger.warning(
+                "Angles are not supported by GRPC in AEDT versions lower than 2024.1."
```

Comment:
```suggestion
                "Angles are not supported by gRPC in AEDT versions lower than 2024 R1."
```
User-facing documentation should use a release format like this: 2024 R1.